types/browser-util-inspect/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for browser-util-inspect 0.2
+// Project: https://github.com/deecewan/browser-util-inspect
+// Definitions by: William So <https://github.com/polyipseity>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = inspect;
+
+declare function inspect(obj: unknown, opts?: inspect.Options): string;
+
+declare namespace inspect {
```

Comment:
Looking at the source, this seems to be missing properties `inspect.colors` and `inspect.styles`, but I can't tell if those are intended for public use or not. This is not a blocking thing, but you still might want to take a look.

Reply:
I couldn't tell whether they are intended to be public too, so I didn't add them in originally. But since they seem to be useful for changing the default style slightly without duplicating the library's code, so I just added them in f5b64c5c5b96dff044353b1bc07ead3aa9320987.